### DISCUSSION TITLE
R# Identifier and its uses

### DISCUSSION THREAD

**Discussion by Thewsthews** [community/community#141432] [Programming Help]
### Select Topic Area

Question

### Body

I recently used the r# raw identifier on a project. The intent was to make the codebase future proof but on compilation, im receiving compilation errors

How do i go about it?

The link to the project is: [https://github.com/Thewsthews/Tasker.git](url)

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @Thewsthews ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
